docs/dev-guides/destroy_deployment.md

```diff
@@ -0,0 +1,227 @@
+
+1. `<deployment_name>-auth` stack (if applicable)
+2. `<deployment_name>-jupyterhub` stack
+3. `<deployment_name>-cluster` stack
+4. *`<deployment_name>-cluster-pipeline` stack
+5. *`<deployment_name>` stack (if applicable)
+
+_NB:*These stacks have additional steps._
+
+**WARNING: As mentioned earlier, do NOT delete the `<deployment_name>-cognito` stack.**
+
+In above order, follow these steps (except for stack 4 and 5):
+
+1. Delete the `<deployment_name>-<stack_name>` CloudFormation stack
+    1. Navigate to the AWS CloudFormation console
+        1. Click the box next to the `<deployment_name>-<stack_name>` stack
+        1. Click the `Delete` button
+                1. Click the `Delete stack` button
+        1. Click the `<deployment_name>-<stack_name>` stack name
+            1. Click the `Events` tab
+            1. Monitor the stack deletion progress
+                1. Click the refresh button periodically since the console doesn't update events automatically
+
+
+If you are deleting stack 4 (`<deployment_name>-cluster-pipeline`), you will need to follow these steps first before deleting the stack:
+
+
+1. Navigate to the AWS S3 console
+    1. If exists, click the `<region>-<deployment_name>-lambda` S3 bucket option.
+    1. Click the `Empty` button
+        1. Confirm the deletion of bucket contents by typing `permanently delete` in the provided field
+        1. Click the `Empty` button
+
+If you are deleting stack 5 (`<deployment_name>`), you will need to follow these steps first before deleting the stack:
+
+1. Delete the `deployment_name` CloudFormation stack
+    1. Delete hub and notifications ECR repos
+        1. Navigate to the AWS Elastic Container Registry
+        1. Click the box next to the `deployment_name`/hub repository
+        1. Click the `Delete` button
+            1. Confirm the deletion by typing "delete" in the provided field
+            1. Click the `Delete` button 
+        1. Repeat the above steps for the `deployment_name`/notifications repository
+    1. Empty the `codepipeline-<region>-<deployment_name>` S3 bucket
+        1. Navigate to the AWS S3 console
+            1. Check the box next to the `codepipeline-<region>-<deployment_name>` S3 bucket
+            1. Click the `Empty` button
+                1. Confirm the deletion of bucket contents by typing `permanently delete` in the provided field
+                1. Click the `Empty` button
+---
+
+## **Delete EBS Snapshots and Volumes**
+
+To mitigate the cost associated with storage space, it is crucial to deallocate unused resources. The below steps will guide you on how to do so.
+
+---
+
+First, navigate to the AWS EC2 console - this step should be identical for both EBS snapshots and EBS volumes.
+
+![ec2 home screen](../assets/docs-ec2-snap-vol.PNG)
+
+### **Delete EBS snapshots**
+1. Click the _Snapshots_ link in the sidebar menu
+    1. Filter by osl-stackname: `<deployment_name>`
```

Comment:
Here and on line 175 `osl-stackname` is not longer used. Properly, it should be `osl-billing`. The idea is that the value of `osl-billing` during deployment creation is unique and used for naming purposes, even if the billing tag is ignored for financial reasons.